### ISSUE TITLE
Can't run ImGui anymore (0xC0000005)

### ISSUE THREAD

**Issue by kylhuk** [AllenDang/giu#140]
Hey there

I had this issue a while ago and I don't know what leads to it. No clue how to debug this error and google does not help at all. The error seems to be quite general.

As soon as I run giu (like the HelloWorld example), I immediately get this error:
```
Process finished with exit code -1073741819 (0xC0000005)
```

Tried different examples. Tried removing the whole giu directory from go src.

What else could it be? Do I need to install special drivers?

OS: Windows 10 20H2
GPU: Nvidia RTX 3090
Go: go1.15.2
IDE: Goland 2020.3

If you need any more infos, please let me know

--
kylhuk

**Comment by AllenDang** [comment]
@kylhuk Can you run it from terminal to see if we can get more error message?

**Comment by kylhuk** [comment]
@AllenDang Ran it in terminal, no different output.

Used `dlv` to debug the code, but it seems not to even get to `main()`. I saw that as soon as I add the include to giu, debugging is not possible anymore in Goland (it just never really starts the debugging process). If I remove that include, everything works fine.

I tested it on a different PC with the exact same result. There I have an Intel graphics card.

The demos of DearImgui run fine.
